LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2019 Talia Ringer, Nate Yazdani
+Copyright (c) 2019 ...
```

Comment:
Lol! My grepping missed that! Will always check upper- and lowercase in the future